contentcuration/contentcuration/views/nodes.py

```diff
@@ -38,7 +36,7 @@ def get_channel_details(request, channel_id):
     channel = get_object_or_404(Channel.filter_view_queryset(Channel.objects.all(), request.user), id=channel_id)
     if not channel.main_tree:
         raise Http404
-    data = get_node_details_cached(channel.main_tree, channel_id=channel_id)
+    data = get_node_details_cached(request.user, channel.main_tree, channel_id=channel_id)
```

Comment:
This view has `AllowAny` permissions, so `request.user` could be an anonymous user here - I do think not letting anonymous users stack our task queue is probably a good thing, but seems like this might raise an error.